lib/ex.coffee

```diff
@@ -71,10 +71,17 @@ class Ex
   @registerCommand: (name, func) =>
     @singleton()[name] = func
 
+  # quit function is used on :q command
   quit: ->
     atom.workspace.getActivePane().destroyActiveItem()
-
+  # quit function is used on :qa command shortcut
+  quitAll: ->
```

Comment:
In vim, this command is named `:quitall`. Could you rename it to that please? (And to the same in line 84)